receiver/prometheusreceiver/internal/metricsbuilder.go

```diff
@@ -98,7 +98,7 @@ func (b *metricBuilder) AddDataPoint(ls labels.Labels, t int64, v float64) error
 				b.scrapeStatus = scrapeStatusOk
 			} else {
 				b.scrapeStatus = scrapeStatusErr
-				b.logger.Warnw("http client error:", "ts", t, "value", v, "labels", lm)
+				b.logger.Warn("http client error", zap.Int64("timestamp", t), zap.Float64("value", v), zap.String("labels", fmt.Sprintf("%v", lm)))
```

Comment:
Zap has a Time field that could possible b used. https://godoc.org/go.uber.org/zap#Time

Reply:
I'd prefer leave it as the raw data as received rather than convert it into time.Time but I can do the conversion if you prefer.

Reply:
I overlooked it and thought it was a time.Time value. Raw definitely makes sense.